src/main/java/com/itschool/eventmanagment/config/WebMvcConfig.java

```diff
@@ -0,0 +1,13 @@
+package com.itschool.eventmanagment.config;
+
+import org.springframework.context.annotation.Configuration;
+import org.springframework.format.FormatterRegistry;
+import org.springframework.web.servlet.config.annotation.WebMvcConfigurer;
+
+@Configuration
+public class WebMvcConfig implements WebMvcConfigurer {
```

Comment:
If you remove the custom converter as suggested, the WebMvcConfig class adding the converter can also be removed. Spring’s `@DateTimeFormat` annotation should handle the date conversion without additional configuration.